js/apps/admin-ui/src/user/UserForm.tsx

```diff
@@ -93,12 +94,13 @@ export const UserForm = ({
   const { realm: realmName } = useRealm();
   const formatDate = useFormatDate();
   const isFeatureEnabled = useIsFeatureEnabled();
-
+  const { config } = useUserProfile();
```

Comment:
Any concerns in regards to this comment: https://github.com/keycloak/keycloak/issues/23507#issuecomment-1734252542 

Do user profile configurations like e.g. "read-only", only user managed, validators, etc. apply?

Reply:
Yeah, this is something we're figuring out, as it's also highly dependent on your user role whether you can see this or not.